src/components/add-form/index.js

```diff
@@ -4,10 +4,8 @@ import './styles.css';
 
 const AddForm = ({ setNewItem, formState, handleForm }) => {
   const [value, setValue] = useState('');
-
   const handleKeyDown = (event) => {
```

Comment:
If you have a button type `submit`, the[ form default behavior](https://tutorial.eyehunts.com/html/html-form-submit-on-enter-key-example-code/) should be submitted when you press `enter`. I believe it's not necessary to `handleKeyDown` function. Can you please check it?

Bonus, only comments 😃 . I think it could be a good idea to [prevent](https://www.tjvantoll.com/2013/01/01/enter-should-submit-forms-stop-messing-with-that/) the `enter` key because you can press `enter` with bad data.